Content.Shared/Access/Systems/AccessReaderSystem.cs

```diff
@@ -115,13 +117,13 @@ public bool IsAllowed(EntityUid user, EntityUid target, AccessReaderComponent? r
         var access = FindAccessTags(user, accessSources);
         FindStationRecordKeys(user, out var stationKeys, accessSources);
 
-        if (IsAllowed(access, stationKeys, target, reader))
-        {
+        if (!IsAllowed(access, stationKeys, target, reader))
+            return false;
+
+        if (!_tag.HasTag(user, "PreventAccessLogging"))
```

Comment:
This string should be put in a `private static readonly ProtoId<TagPrototype>` so that it can statically verified.